release-notes/3.0/preview/api-diff/preview4/standalone-packages/3.0-preview4-standalone-packages.md

```diff
@@ -0,0 +1,13 @@
+# API Difference netcoreapp3.0-preview3 vs netcoreapp3.0-preview4
```

Comment:
Similar feedback as the other PR: https://github.com/dotnet/core/pull/2638#discussion_r279561069